research-domain-agent/src/test/java/care/smith/fts/rda/rest/it/GeneralIT.java

```diff
@@ -136,4 +137,37 @@ void callingStatusReturnsQueued() throws IOException {
                     .verifyThenAssertThat()
                     .hasOperatorErrors());
   }
+
+  @Test
+  void projects() {
+    StepVerifier.create(client.get().uri("/api/v2/projects").retrieve().bodyToMono(String.class))
+        .assertNext(
+            transferProcessDefinitions -> {
+              assertThat(transferProcessDefinitions).isEqualTo("[\"test\"]");
+            })
+        .verifyComplete();
+  }
+
+  @Test
+  void project() {
+    StepVerifier.create(
+            client.get().uri("/api/v2/projects/test").retrieve().bodyToMono(String.class))
+        .assertNext(
+            transferProcessDefinitions -> {
+              assertThat(transferProcessDefinitions).contains("trustCenterAgent");
+              assertThat(transferProcessDefinitions).contains("MII");
```

Comment:
Have you checked if we can assert parts of the json, instead of just searching in the string?

Reply:
There is this: https://github.com/lukas-krecan/JsonUnit
But it introduces unnecessary complexity for this use case, and might not even work because in the case of *TrustCenterAgent* we just want to check that the key exists somewhere in the JSON.